src/libos/src/vm/process_vm.rs

```diff
@@ -73,6 +91,7 @@ impl<'a, 'b> ProcessVMBuilder<'a, 'b> {
                     })
             })
             .collect();
+
         let other_layouts = vec![
             VMLayout::new(heap_size, PAGE_SIZE)?,
```

Comment:
We may use a smaller alignment value in the feature implementation.  Please add a "Todo" here.

Reply:
Done. Thank you for your review.